src/utils/i18n.js

```diff
@@ -83,6 +85,7 @@ export const resources = {
       balance: "잔액",
       from: "보내는이",
       to: "받는이",
+      your_eth_address: "귀하의 이더 리움 주소",
```

Comment:
I didnt know you know korean as well :P

Reply:
hahah not really. Just to fill something there. Maybe later somebody could review it or we can also remove it